src/test/acceptance/features/privacy-notice.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Privacy notice page
+  In order to facilitate the beta HTBHF programme
+  As a potential claimant
+  I want to see the details of how my details are used and retained
+
+  Scenario: The Privacy notice page is accessible
+    Given I navigate to the HTBHF overview page
```

Comment:
Might it be prudent for this to not be the Overview page and rather the enter name page as the Overview page will disappear at some point?

Reply:
Fair point. There is then no difference between this and the last scenario - shall I remove this one? The same would apply for the cookies feature.

Reply:
Yes, I believe so.